src/doc/functional/_enrolment.rst

```diff
@@ -2,4 +2,197 @@
 Enrolment Services
 ------------------
 
-:todo:`To be defined`
+
+
+Services
+""""""""
+
+.. py:function:: createPerson(personID, personData, options)
+    :noindex:
+
+    Insert a new person.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str personID: The ID of the person. If the person already exists for the ID an error is returned.
+    :param dict personData: The person attributes.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error.
+
+.. py:function:: getPerson(personID, options)
+    :noindex:
+
+    Retrieve the attributes of a person.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str personID: The ID of the person.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error and in case of success the person data.
+
+.. py:function:: updatePerson(personID, personData, options)
+    :noindex:
+
+    Update a person.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str personID: The ID of the person.
+    :param dict personData: The person data.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error.
+
+.. py:function:: deletePerson(personID, options)
+    :noindex:
+
+    Delete a person and all their identities.
```

Comment:
The _and all their identities_ part looks like a copy/paste error. Can you remove it?